lib/big_query/timestamp_rollup.ex

```diff
@@ -0,0 +1,9 @@
+defmodule BigQuery.TimestampRollup do
+  @callback name() :: String.t
+  @callback rollup() :: String.t
+  @callback is_a?(String.t) :: boolean
+  @callback floor(%DateTime{}) :: %DateTime{}
+  @callback ceiling(%DateTime{}) :: %DateTime{}
+  @callback range(%DateTime{}, %DateTime{}) :: [%DateTime{}]
```

Comment:
could add range w/ 3rd arg here to enforce inclusion?

Reply:
Good idea!  That would make it more explicit what's going on.